Model/Token.php

```diff
@@ -74,12 +83,17 @@ public function getToken()
 
             $result = json_decode(file_get_contents($certCaptureConfig['url'], false, $context), true);
 
-            return [
+            return $this->tokenInterfaceFactory->create(
                 [
-                    'token' => $result['response']['token'],
-                    'expires' => (new \DateTime($result['response']['expires_at']))->getTimestamp()
+                    'data' => [
+                        'token' => $result['response']['token'],
```

Comment:
This code and the lines below it assume the existence of the array keys, which could cause E_NOTICEs. Change to something like:
```
'token' => $result['response']['token'] ?? null,
```

Reply:
There is a contract promising the existence though. The cert capture config doesn't have the same contract though, as that is user-provided data. If those don't exist, however, it would all break anyway. Therefore, I agree, on at least `certCaptureConfig` at least, that there could be a pre-check and throw an exception if they don't exist. Thoughts?